app/views/sign_up/passwords/new.html.erb

```diff
@@ -7,17 +7,13 @@
 </p>
 <%= simple_form_for(
       @password_form,
+      as: :password_form,
       url: sign_up_create_password_path,
       method: :post,
       html: { autocomplete: 'off' },
     ) do |f| %>
-  <%= render PasswordToggleComponent.new(
+  <%= render PasswordConfirmationComponent.new(
         form: f,
-        field_options: {
-          label: t('forms.password'),
-          required: true,
-          input_html: { aria: { describedby: 'password-description' } },
```

Comment:
Do we want to maintain the `aria-describedby` in the new implementation? Or, if not, should the existing elements with `id="password-description"` have their IDs removed?

Reply:
Added `aria-describedby`.